docs/workshop-hello-world.md

```diff
@@ -0,0 +1,261 @@
+# Purpose
+This is meant to be a small workshop that explores deploying a hello world application
+to the AWS EKS Kubernetes cluster, as well as exploring a few items available within
+the cluster.
+
+## Pre-Requisites
+- Access to the AWS account that an EKS cluster is deployed to. In this workshop we will assume you are using the `org-sagebase-dnt-dev` account.
+- You will need either `Administrator` or `Developer` access in the related AWS account. Both will have a level of access to the EKS cluster. In this workshop we are assuming the use of `Administrator` access.
+- Install a tool to access the cluster in a user friendly way such as: https://k9scli.io/
+- Set up a SSO login session to access the cluster and login, for example: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2632286259/AWS+SSM+Session+Manager#%5CuD83D%5CuDCD8-Log-in-to-an-AWS-account
+  - `aws sso login --profile dnt-dev-admin`
+  - `aws sso login --profile dnt-dev-developer`
+- Update your [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) to connect to the cluster. The kubeconfig in this case is referring to the authentication mechanism used to access the cluster.:
+    - `aws eks update-kubeconfig --region us-east-1 --name dpe-k8-sandbox --profile dnt-dev-admin`
+    - `aws eks update-kubeconfig --region us-east-1 --name dpe-k8-sandbox --profile dnt-dev-developer`
+- In your cli start k9s
+    - `k9s`
+
+If everything is correct you should see a screen like:
+![k9s header](./workshop-resources/k9s-header.png)
+
+
+### Deploying the resources through ArgoCD (Recommended)
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes. What this
+means is that it will take care of all the deployment of the resources for you. For our
+use case we will be creating an 
+[Application Specification](https://argo-cd.readthedocs.io/en/stable/user-guide/application-specification/).
+This Application Specification defines a number of pointers and configuration elements
+for ArgoCD to deploy to the Kubernetes cluster. Once ArgoCD has deployed the resources
+you can view the resources in the ArgoCD UI, as well as in the k9s UI.
+
+
+- Create a new branch in git to push your changes to, for example `ibcdpe-1-branch-name`
+- Open up `deployments/stacks/dpe-k8s-deployments/main.tf`
+  - You'll notice a bunch of modules. These are the resources we are already deploying to the kubernetes cluster, wrapped in something called a "module"
+  - For this tutorial we will create the resource files directly here for simplicity
+
+- Create 3 sections in the file:
+
+1) A locals block: <https://www.terraform.io/docs/configuration/locals.html> to organize our inputs
+
+```terraform
+locals {
+  my_branch_name = "ibcdpe-1-branch-name"
```

Comment:
Add a note to rename what's in `my_branch_name` to whatever the users' branch name is